src/libraries/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs

```diff
@@ -341,6 +341,12 @@ public void Close(int timeout)
             Dispose();
         }
 
+        public void Close(TimeSpan timeout)
+        {
+            int milliseconds = (int)timeout.TotalMilliseconds;
+            Close(milliseconds);
+        }
```

Comment:
```suggestion
        public void Close(TimeSpan timeout) => Close(checked((int)timeout.TotalMilliseconds))
```

At least, I assume we want checked here.